lib/diff.js

```diff
@@ -87,7 +87,12 @@ var operations = {
     /*
      * { command: 'setTransition', args: [transition] }
      */
-    setTransition: 'setTransition'
+    setTransition: 'setTransition',
+
+    /*
+     * { command: 'setLighting', args: [lightProperties] }
+     */
+    setLighting: 'setLighting'
```

Comment:
It looks like we use the name `light` rather than `lighting` elsewhere to refer to light properties. What do you think about changing this name to `setLight`?